modules/platform/nuxeo-automation/nuxeo-automation-features/src/main/java/org/nuxeo/ecm/automation/core/operations/services/bulk/AutomationBulkActionUi.java

```diff
@@ -26,7 +26,7 @@
  *
  * @since 11.5
  */
-public class AutomationBulkActionUi extends AutomationBulkAction {
+public class AutomationBulkActionUi extends AbstractAutomationBulkAction {
```

Comment:
is this wanted?

Reply:
Yes it is, otherwise, the abstract is not useful and we could merge `AutomationBulkAction` & `AbstractAutomationBulkAction`.